web/src/pages/TraceDetailPage.tsx

```diff
@@ -40,22 +42,26 @@ export function TraceDetailPage() {
           <Stack>
             <Breadcrumb>
               <BreadcrumbItem>
-                <Link to="/observe/traces">{t('Traces')}</Link>
+                <Link to={`/observe/traces${location.search}`}>
```

Comment:
otherwise the current TempoStack selection is reset

Reply:
This is fine for now, in a follow up PR we could add a hook that handles several parameters from the URL, like the current query, selected tempo stack, time range etc.

Reply:
:+1:
I'll include a hook for this in the next PR, where I also need this functionality for the tempo {namespace, name, tenant} selection.